### ISSUE TITLE
E206 fails when multiline

### ISSUE THREAD

**Issue by juan-acevedo-ntt** [ansible-community/ansible-lint#669]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible: 2.9.1
ansible-lint: 4.2.0
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour

An error should be raised as there is no space before the double curly braces:
[E206] Variables should have spaces before and after: {{ var_name }}

This was working OK in v.4.1.0

# Actual Behaviour

Ansible task:
```
    - name: 2-3.9-7 - Log error if PidFile has wrong permissions
      set_fact:
        compliance_apache_pidfile_errors: >
          {{ compliance_apache_pidfile_errors
          + ['Must be owned by root user and group,
          and only writable by root']}}
      when: compliance_apache_pidfile_permissions.changed
```

The ansible lint shows no error, but a space befere "}}" should be present. This is bug is only present when the jinja expression is in multiple lines.

**Comment by ssbarnea** [comment]
Fixed on master, 5.0.0 will include it.
